lib/src/screens/home_screen.dart

```diff
@@ -1,10 +1,34 @@
 import 'package:flutter/material.dart';
 
+import '../constants/constants.dart' as constants;
+
 class HomeScreen extends StatelessWidget {
-  const HomeScreen({Key? key}) : super(key: key);
+  const HomeScreen({ Key? key, this.onPressed}) : super(key: key);
+  final VoidCallback? onPressed;
 
   @override
   Widget build(BuildContext context) {
-    return const Text('HomeScreen');
+    double logoSize = MediaQuery.of(context).size.height * .13;
+    double toolBarSize = MediaQuery.of(context).size.height * .11;
+    double iconSize = MediaQuery.of(context).size.height * .04;
+
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
Not a request but we should consider if this is something we want to turn into a common widget for reusability and less instanced variables in the screen file

Reply:
I thought through this case, it's similar to Ms. Van's, but Ms. Van's AppBar is changed according to the input variable, so it can be reused a lot. As for my AppBar, it's , so I don't think it's necessary to turn into a common widget.